pkg/util/logs.go

```diff
@@ -0,0 +1,13 @@
+package util
+
+const (
+	Debug     = "debug"
```

Comment:
These should either be moved to a less generic package or named less generically. Right now they're called like so from elsewhere: `util.Instance`, which is not clear. It would be typical to name them like so: `LogLevelInfo`, and `LogScopeInstance`.